tests/lxd/test_lxd_backend.cpp

```diff
@@ -653,6 +654,65 @@ TEST_F(LXDBackend, machine_does_not_update_state_in_dtor)
     EXPECT_TRUE(stop_requested);
 }
 
+TEST_F(LXDBackend, machine_logs_not_found_exception_in_dtor)
```

Comment:
Could you use camelCase for this? We have a bunch of snake_cased tests still, but we're trying to move from it because gtest advises against underscores in test names (it can mess with how it uses underscores to generate internal test names).